physionet-django/console/forms.py

```diff
@@ -482,13 +482,18 @@ def save(self):
         affiliation = super().save(commit=False)
         display_order = self.project.authors.count() + 1
         submitting = False
+        is_corresponding = False
+        corresponding_email = None
         if display_order == 1:
             submitting = True
+            is_corresponding = True
+            corresponding_email = author.email
```

Comment:
`corresponding_email` doesn't seem to get saved anywhere?

Reply:
Good catch :D I just added it to the published author creation